### PR TITLE
Always keep first font

### PR DESCRIPTION
**Pull request by dogusev**
Font skipped each time, when `styleId===0`
related issues
https://github.com/exceljs/exceljs/issues/849
https://github.com/exceljs/exceljs/issues/833
https://github.com/exceljs/exceljs/issues/816

### CODE REVIEW

**Comment by dogusev** [comment]
@guyonroche Hi!
I found a serious issue, which causes a lot of differences in documents styles.
The default font was always removed.
Please merge this PR ASAP

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "exceljs",
-  "version": "1.12.1",
+  "version": "1.12.2",
```
Comment (guyonroche) [inline package.json]: Please don't edit this - it is managed during the release/publish process

File: spec/utils/test-spliced-sheet.js [hunk 2]
```diff
@@ -227,30 +227,24 @@ module.exports = {
         expect(ws.getRow(2).values).to.deep.equal([, 'one', 'two', 'three']);
         expect(ws.getRow(3).values).to.deep.equal([, '2,1', '2,2', '2,3']);
 
-        expect(ws.getRow(3).style).to.deep.equal({
-          alignment: {
-            horizontal: 'left',
-            vertical: 'middle',
-          },
+        expect(ws.getRow(3).style.alignment).to.deep.equal({
```
Comment (guyonroche) [inline spec/utils/test-spliced-sheet.js]: Could you add a test that covers your change?

File: package.json (hunk 1 above)
Comment (dogusev) [reply package.json]: ok

File: spec/utils/test-spliced-sheet.js (hunk 2 above)
Comment (dogusev) [reply spec/utils/test-spliced-sheet.js]: it took o lot of time to find the issue, not sure i'll be able cover with tests coming days

**Comment by elenkali** [comment]
This change seems to break all our formating. How do I override it for other rows now?

**Pull request merged**